cli/cli.ts

```diff
@@ -33,6 +33,7 @@ const rimraf: (f: string, opts: any, cb: (err: any, res: any) => void) => void =
 let forceCloudBuild = process.env["KS_FORCE_CLOUD"] !== "no";
 let forceLocalBuild = !!process.env["PXT_FORCE_LOCAL"];
 let forceBuild = false; // don't use cache
+let doPythonRoundTripSyntaxComparison = false
```

Comment:
let's avoid globals in the CLI. get this argument from the parsed arguments where you need it.